src/connectedk8s/azext_connectedk8s/_params.py

```diff
@@ -47,6 +50,11 @@ def load_arguments(self, _):
         c.argument('proxy_cert', options_list=['--proxy-cert'], arg_group='Proxy', type=file_type, completer=FilesCompleter(), help='Path to the certificate file for proxy')
         c.argument('disable_proxy', options_list=['--disable-proxy'], arg_group='Proxy', action='store_true', help='Disables proxy settings for agents')
         c.argument('auto_upgrade', options_list=['--auto-upgrade'], help='Flag to enable/disable auto upgrade of arc agents. By default, auto upgrade of agents is enabled.', arg_type=get_enum_type(["true", "false"]))
+        c.argument('container_registry_repository', options_list=['--pcr-repository'], help='Custom container registry repository.')
+        c.argument('container_registry_username', options_list=['--pcr-username'], help='Custom container registry repository username.')
+        c.argument('container_registry_password', options_list=['--pcr-password'], help='Custom container registry repository password.')
+        c.argument('default_registry', options_list=['--default-registry'], action='store_true', help='Flag to restore default container registry (MCR).')
+        c.argument('anonymous_pull', options_list=['--anonymous-pull'], action='store_true', help='Flag to change PCR with credentials to PCR with anonymous pull.')
```

Comment:
we do not need an additional parameter for this. as per the PM spec, if the customer provides username and password values, we create imagePullSecret. If not provided, we just assume it's anonymous pull and deploy agent charts without imagePullSecrets.

Reply:
as discussed  we will now have --use-default-registry and --anonymous-pull